File: backend/Models/Nation.cs

using System.Collections.Generic;

namespace annexio.models
{
    public class Nation
    {
        public NationName Name { get; set; }
        public int Population { get; set; }
        public object Currencies { get; set; }


        public List<string> Capital { get; set; }

        public object Languages { get; set; }

        public string Cca3 { get; set; }
        public string Cca2 { get; set; }
    }
}

File: frontend/src/Models/Nation.ts
import { Flags } from "./Flags";
import { NationName } from "./NationName";

export interface Nation {

    name: NationName;
    population: number;
    currencies: object;
    capital: string[];
    languages: object;
    cca3: string
    cca2: string;
    flags: Flags;
}